app/src/main/assets/succes.html

```diff
@@ -0,0 +1 @@
+<strong id="test">TEST PAGE</strong>
```

Comment:
could this file be in the test package? I wouldn't want to mix production assets with test assets if possible

Reply:
Unfortunately they have to be in the assets folder but I made a test sub-directory within assets to fix it.

Reply:
clear enough.